app/support/accessories/accessorizer.rb

```diff
@@ -110,6 +110,7 @@ def detail_attributes(accessory, options)
     attrs = @order_detail.attributes.slice("account_id", "created_by")
     attrs.merge(order: @order_detail.order,
                 product: accessory,
+                note: options[:note],
```

Comment:
I decided to allow this so that it was easier to debug the tests. I don't think there's any harm in allowing a note field on an accessory order detail.